tests/init.test.js

```diff
@@ -1,49 +1,103 @@
 const utils = require('./utils');
-const testRepo = require('./utils').testRepo;
 const fs = require('fs-extra');
 const path = require('path');
 const yaml = require('js-yaml');
 
-const { EXPLAIN_PLACEHOLDER } = require('../utils/common');
+const { EXPLAIN_PLACEHOLDER, shouldBeCollapsed } = require('../utils/common');
 
 // Tmp directories used in tests.
 let tmpDirs = Array();
 
 describe('tuture init', () => {
 
   afterAll(() => {
-    tmpDirs.map(dir => fs.removeSync(dir));
+    tmpDirs.forEach(dir => fs.removeSync(dir));
     process.chdir(path.join(__dirname, '..'));
   });
 
-  // TODO: Test prompts when no args given.
+  describe('no .gitignore', () => {
```

Comment:
I'd like naming like this: 

- Git repo exists: `test content`

- Git repo not exists: `test content`

  eg: `test content`
  - no `.gitignore`
  - .gitignore already having .tuture ignored